### PR TITLE
#3-Semantic-HTML-Meysam-Arshadi

### LINKED ISSUE #3: Create the semantic HTML
- [x] Modified the existing `index.html` file located in the `src` folder of the React app.
- [x] Added the `<header>`, `<main>`, and `<footer> `elements to the `index.html` file to create a basic semantic structure.
- [x] Updated the page title inside the `<head>` section to <`title>CYF Jobs Board</title>`.

### PR DESCRIPTION
**Pull request by Meysam-Arshadi**
Fixed #3 
- [x] Modified the existing `index.html` file located in the `src` folder of the React app.
- [x] Added the `<header>`, `<main>`, and `<footer> `elements to the `index.html` file to create a basic semantic structure.
- [x] Updated the page title inside the `<head>` section to <`title>CYF Jobs Board</title>`.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/index.html [hunk 1]
```diff
@@ -3,15 +3,23 @@
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="initial-scale=1, width=device-width" />
-		<title>Starter Kit</title>
+		<title>CYF Jobs Board</title>
```
Comment (OlushoreOdude) [inline client/src/index.html:6]: looks good

File: client/src/index.html [hunk 2]
```diff
@@ -3,15 +3,23 @@
...
+			<div id="root"></div>
+			<script
+				crossorigin
+				type="text/javascript"
+				src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es6"
+			></script>
+		</main>
+		<footer>
+			<!-- Footer content goes here -->
+		</footer>
```
Comment (OlushoreOdude) [inline client/src/index.html:23]: @Meysam-Arshadi, I think Oliver was referring to the layout component  not the root.

**Review by OlushoreOdude — CHANGES_REQUESTED** [review]
@Meysam-Arshadi, title change good, remove header main and footer from root and create a page layout Component with sematic html in src folder.

File: client/src/index.html (hunk 2 above)
Comment (Meysam-Arshadi) [reply client/src/index.html]: Thanks @OlushoreOdude. Done!

**Pull request merged**